### PR TITLE
HOTFIX: Fix compile error in TopicAdminTest

### PR DESCRIPTION
**Pull request by chia7712**
9a4f00f78bf37041006ae8b6432d194f603ac6cc changed the constructor of DescribeConfigsResponse.

```
/home/chia7712/kafka/connect/runtime/src/test/java/org/apache/kafka/connect/util/TopicAdminTest.java:582: error: incompatible types: int cannot be converted to Struct
        return new DescribeConfigsResponse(1000, configs);
                                           ^
Note: Some messages have been simplified; recompile with -Xdiags:verbose to get full output
1 error
```

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by chia7712** [comment]
@tombentley Could you take a look?

**Review by abbccdda — COMMENTED** [review]
@chia7712 Could you include the actual failure in the summary?

File: connect/runtime/src/test/java/org/apache/kafka/connect/util/TopicAdminTest.java [hunk 1]
```diff
@@ -568,18 +570,20 @@ private DescribeConfigsResponse describeConfigsResponseWithTopicAuthorizationExc
 
     private DescribeConfigsResponse describeConfigsResponse(ApiError error, NewTopic... topics) {
         if (error == null) error = new ApiError(Errors.NONE, "");
-        Map<ConfigResource, DescribeConfigsResponse.Config> configs = new HashMap<>();
+        List<DescribeConfigsResponseData.DescribeConfigsResult> results = new ArrayList<>();
         for (NewTopic topic : topics) {
```
Comment (kkonstantine) [inline connect/runtime/src/test/java/org/apache/kafka/connect/util/TopicAdminTest.java]: let's stream the topics and use a collector in the end.

File: connect/runtime/src/test/java/org/apache/kafka/connect/util/TopicAdminTest.java [hunk 2]
```diff
@@ -568,18 +570,20 @@ private DescribeConfigsResponse describeConfigsResponseWithTopicAuthorizationExc
...
+                    .setErrorCode(error.error().code())
+                    .setErrorMessage(error.message())
+                    .setResourceType(ConfigResource.Type.TOPIC.id())
+                    .setResourceName(topic.name())
+                    .setConfigs(topic.configs().entrySet().stream().map(e -> new DescribeConfigsResponseData.DescribeConfigsResourceResult()
+                            .setName(e.getKey())
+                            .setValue(e.getValue())).collect(Collectors.toList())));
+        }
+        return new DescribeConfigsResponse(new DescribeConfigsResponseData()
+        .setThrottleTimeMs(1000)
```
Comment (kkonstantine) [inline connect/runtime/src/test/java/org/apache/kafka/connect/util/TopicAdminTest.java]: nit: indentation is off here. You may break at the second `new` and have the argument in one line probably

**Review by kkonstantine — COMMENTED** [review]
Thanks for the quick fix @chia7712 
I left a couple of comments.

**Comment by chia7712** [comment]
@abbccdda I attache the error message to the description.

@kkonstantine I have addressed your comment

Please take a look :)

**Review by kkonstantine — APPROVED** [review]
Thanks @chia7712 
LGTM

Waiting for a green build

**Comment by tombentley** [comment]
Sorry about breaking the build. Thanks @chia7712.

**Review by tombentley — APPROVED** [review]
LGTM

**Comment by kkonstantine** [comment]
The few test failures were not relevant. Merging to fix the build anyways.

**Pull request merged**